### ISSUE TITLE
getFile creates parent folders in Google Drive

### ISSUE THREAD

**Issue by kevincox** [remotestorage/remotestorage.js#1186]
See the following example:

```js
let client = remoteStorage.remote.rs.scope("/com.example/");
client.getFile("some/folder/that/doesnt/exist");
```
This will create `some/folder/that/doesnt`. This is incredibly surprising for a read operation and can result in some applications that rely on folder listing to fail as now there are empty folders without the expected contents.

**Comment by michielbdejong** [comment]
Right, it shouldn't be necessary to create parent folders in situations where the file itself does not get created.

**Comment by sorentycho** [comment]
This also seems in some circumstances to be able to cause the creation multiple copies of the nonexistent parent directory, if there are multiple requests in flight at once.
